### ISSUE TITLE
Umm... What Just Happened? Need Help Setting Up Unimined

### ISSUE THREAD

**Issue by pvpb0t** [WagYourOrg/unimined#3]
I was browsing the web and suddenly I was redirected to your page. I have no idea what's going on, but I was hoping you could help me set it up? I mean, I clicked on a link and now I'm here. I hope this makes sense to you, because it sure doesn't to me!🎮💻Can you explain to me in layman's terms what Unimined is and how I can use it to mine some serious laughs and good times? I mean, I've heard rumors about a unified minecraft modding environment, but I have no idea what that means. Can I use it to mine chocolate or unicorn tears or something like that?💔
Thanks for your time and I'm looking forward to hearing back from you soon!

**Comment by wagyourtail** [comment]
It supports multiple modding environments in a similar structure to arch-loom, the difference being that unimined supports older forge versions as well. For an example of how to set this up, see https://github.com/jsmacros/jsmacros 's `build.gradle`s

**Comment by pvpb0t** [comment]
This is not fair... close without answer
